languages/csharp/reference/exercise-concepts/gigasecond.md

```diff
@@ -0,0 +1,40 @@
+# Concepts of gigasecond
+
+[Example implementation](https://github.com/exercism/csharp/blob/master/exercises/gigasecond/Example.cs)
+
+## General concepts
+
+### Object-oriented
```

Comment:
We've simplified the format lately, and are doing away with the paradigm divisions.

```suggestion
```